src/refactoring/session-manager-simple.ts

```
@@ -0,0 +1,367 @@
+        name,
+        description,
+        status: 'active',
+        target_branch: targetBranch || '',
+        start_time: Date.now(),
+        metadata: {},
+        created_at: now,
+        updated_at: now
+      };
+
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)